modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -42,81 +37,50 @@ The output verifies that the `ServiceBinding` CR is created to project the bindi
 +
 .Example output
 ----
-servicebinding.binding.operators.coreos.com/spring-petclinic-rest created
+servicebinding.binding.operators.coreos.com/spring-petclinic created
 ----
 
-. To verify that the binding is successful, check the status conditions of the binding resource:
+. Verify that the request for service binding is successful:
```

Comment:
Should verification steps like this on have their own separate verification section, [like the one shown in the template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc)?
```suggestion
.Verification

. Verify that the request for service binding is successful:
```

Reply:
Thanks, @rolfedh for the suggestion. Ideally yes, but this QSG has multiple procedures and some have verification steps for each step, so to maintain consistency I prefer not to use a separate Verification section for now, but to keep it aligned with every step as required.